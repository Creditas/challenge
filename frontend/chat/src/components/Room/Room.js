import React, { Fragment } from 'react';
import Tabs from '@material-ui/core/Tabs';
import Tab from '@material-ui/core/Tab';

import styles from './room.css';

const Room = ({children}) => (
  <Fragment>
    <Tabs value={0} className={styles.wrapper}>
      <Tab label="Room 1" />
    </Tabs>
    {children}
  </Fragment>
)

export default Room;
