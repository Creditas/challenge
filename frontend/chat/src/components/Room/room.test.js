import React from 'react';
import { shallow } from 'enzyme';
import Room from './Room';

describe('Room', () => {
  it('should render', () => {
    const rendered = shallow(<Room />);
    expect(rendered).toMatchSnapshot()
  })
})
