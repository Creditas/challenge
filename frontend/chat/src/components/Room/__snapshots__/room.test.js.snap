// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Room should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Room />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <WithStyles(Tabs)
          className="wrapper"
          value={0}
>
          <WithStyles(Tab)
                    label="Room 1"
          />
</WithStyles(Tabs)>,
        undefined,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(Tab)
            label="Room 1"
/>,
          "className": "wrapper",
          "value": 0,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "label": "Room 1",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      undefined,
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <WithStyles(Tabs)
            className="wrapper"
            value={0}
>
            <WithStyles(Tab)
                        label="Room 1"
            />
</WithStyles(Tabs)>,
          undefined,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <WithStyles(Tab)
              label="Room 1"
/>,
            "className": "wrapper",
            "value": 0,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "label": "Room 1",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        undefined,
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
